src/mesh/checkpoint_io.C

```diff
@@ -76,6 +78,53 @@ void chunking(processor_id_type size, processor_id_type rank, unsigned int nspli
                              (1 - (int)nchunks) * std::numeric_limits<int>::max());
     }
 }
+
```

Comment:
I just realized that this anonymous namespace is declared *inside* of `namespace libMesh`... I'm not sure how this works, are these functions actually local only to this file, or are they actually in namespace libMesh? Maybe just move the opening `namespace libMesh {` down to avoid this potential confusion.